modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -1454,6 +1454,8 @@
 			var eventHandles = [Liferay.once(config.eventName, callback)];
 
 			var detachSelectionOnHideFn = function(event) {
+				Liferay.fire(config.eventName);
```

Comment:
Ideally, we should only fire the event if it wasn't fired yet, right? Therefore, let's add a checking like the one below:
```
if (!eventHandles[0].evt.fired) {
    Liferay.fire(config.eventName);
}
```

Reply:
I disagree in this case. Following the steps described in LPS-105059, let's add these ones:

7. Realize you also want to have a template called "Adios". So click on "Manage Templates" link and the modal is opened again.
8. Create the new "Adios" template
9. Close the modal
10. Open "Display Template" combo

Expected Result: You can see both new templates "Hola" and "Adios"
Actual Result: You only can see the first template you saved "Hola" => Because the event is fired only the first time

Reply:
Hi @antonio-ortega,

I made this suggestion, because I was trying to avoid a duplicated firing of the event, which will occur in the other scenarios that the DDMPortlet is opened, I mean the use cases in which the Manage Templates link isn't clicked. However, this use case shared by you can be an issue, thus I'll double check if my suggestion works and then forward your pull.

Thanks.

Reply:
Hey @antonio-ortega,

I just tested the use case shared by you and indeed my last suggestion won't work, therefore I'll just run the relevant tests and if everything is fine, then I'll forward your pull. 🙂 

Thanks.